examples/tcp-retransmit.bpf.c

```diff
@@ -0,0 +1,84 @@
+#include <vmlinux.h>
+#include <bpf/bpf_tracing.h>
+#include <bpf/bpf_core_read.h>
+#include "string.h"
```

Comment:
There is no `string.h` in `examples`, so you probably wanted `<string.h>`. It seems that you're using it for `memcpy`, but my understanding is that you're supposed to use `bpf_probe_read_kernel` instead.

None of this matters if you don't put high cardinality IP addresses into labels.